### ISSUE TITLE
Review ASpaceAPI library for additional ideas

### ISSUE THREAD

**Issue by helrond** [RockefellerArchiveCenter/rac_aspace#15]
Review @djpillen's [ASpaceAPI](https://github.com/djpillen/ASpaceAPI/blob/master/aspaceapi/ASpaceAPI.py) library, particularly lines 499 and higher, to see if there are additional useful functions that can be extracted.

**Comment by djpillen** [comment]
FYI, that client library has since been moved to the [bentley-historical-library/bhlaspaceapiclient](https://github.com/bentley-historical-library/bhlaspaceapiclient/blob/master/bhlaspaceapiclient/client.py) repo.

I don't think the two are a whole lot different, but the Bentley's version will be the most up-to-date.

Feel free to reach out if you have questions about anything in the library. Most of the functions have been tacked on over the years when I realized I was doing the same thing over and over again in different scripts. I would guess that some are broadly useful whereas others are probably serving more idiosyncratic use cases.
